.github/pull_request_template.md

```diff
@@ -6,3 +6,12 @@
 ## Solution
 
 - Describe the solution used to achieve the objective above.
+
+## Changelog
+
+- List a general overview of the changes made by this PR
+- Try to be concise. If more detail is needed for a particular change, consider adding it to the "Solution" section
+
+## Migration Guide
```

Comment:
```suggestion
## \[Optional\] Migration Guide
```

Reply:
Should this be added to Changelog as well?